template/src/ontology/Makefile.jinja2

```diff
@@ -219,13 +219,19 @@ RELEASE_ASSETS = \
   $(REPORT_FILES){% endif %}
 
 .PHONY: all_assets
-all_assets: $(ASSETS)
+all_assets: $(ASSETS) {% if project.ensure_valid_rdfxml %}check_rdfxml_assets{% endif %}
 
 .PHONY: show_assets
 show_assets:
 	echo $(ASSETS)
 	du -sh $(ASSETS)
 
+check_rdfxml_%: %
+	@check-rdfxml {% if project.extra_rdfxml_checks %}--extra{% endif %} $<
```

Comment:
Disable echoing the command before running it. There’s no need here to have Make print a `check-rdfxml file.owl` on the terminal, since the `check-rdfxml` script itself will print a message indicating what it does.